src/routers/__tests__/TabRouter-test.js

```diff
@@ -215,7 +215,6 @@ describe('TabRouter', () => {
     const navAction = {
       type: NavigationActions.NAVIGATE,
       routeName: 'Baz',
-      params: { foo: '42', bar: '43' },
```

Comment:
Specifying params to a `NAVIGATE` call that hits a router is non-sensical, as a router cannot take params. Indeed, the only reason this had any effect in the first place was because of the weird way the removed code behaved.